NPE when calling getResponsiveSteps()

```java
FormLayout formLayout = new FormLayout();
formLayout.getResponsiveSteps();
```
throws NPE:
```
Caused by: java.lang.NullPointerException
	at com.vaadin.flow.component.formlayout.FormLayout.getResponsiveSteps(FormLayout.java:225)
	at org.vaadin.jonni.MainView.<init>(MainView.java:15)
	... 53 more
```
Using Vaadin platform 11.0.1 and vaadin-form-layout-flow-1.0.3.

[Mimimal project to reproduce the problem](https://github.com/vaadin/vaadin-form-layout-flow/files/2460813/form_layout_responsive_steps_npe.zip)

--- a/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java
+++ b/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java
@@ -17,6 +17,7 @@ package com.vaadin.flow.component.formlayout;
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 import java.util.concurrent.atomic.AtomicInteger;
@@ -221,6 +222,9 @@ public class FormLayout extends GeneratedVaadinFormLayout<FormLayout>
     public List<ResponsiveStep> getResponsiveSteps() {
         JsonArray stepsJsonArray = (JsonArray) getElement()
                 .getPropertyRaw("responsiveSteps");
+        if (stepsJsonArray == null) {
+            return Collections.emptyList();
+        }
         List<ResponsiveStep> steps = new ArrayList<>();
         for (int i = 0; i < stepsJsonArray.length(); i++) {
             steps.add(stepsJsonArray.get(i));

--- a/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java
+++ b/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java
@@ -244,7 +244,7 @@ public class FormLayout extends GeneratedVaadinFormLayout<FormLayout>
         AtomicInteger index = new AtomicInteger();
         getElement().setPropertyJson("responsiveSteps",
                 steps.stream().map(ResponsiveStep::toJson).collect(
-                        () -> Json.createArray(),
+                        Json::createArray,
                         (arr, value) -> arr.set(index.getAndIncrement(), value),
                         (arr, arrOther) -> {
                             int startIndex = arr.length();

--- a/src/test/java/com/vaadin/flow/component/formlayout/tests/FormLayoutTest.java
+++ b/src/test/java/com/vaadin/flow/component/formlayout/tests/FormLayoutTest.java
@@ -1,0 +1,30 @@
+/*
+ * Copyright 2000-2018 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.formlayout.tests;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.vaadin.flow.component.formlayout.FormLayout;
+
+public class FormLayoutTest {
+
+    @Test
+    public void getResponsiveSteps_noInitialSteps_emptyListIsReturned() {
+        FormLayout layout = new FormLayout();
+        Assert.assertTrue(layout.getResponsiveSteps().isEmpty());
+    }
+}

